CONTRIBUTING.md

```diff
@@ -208,7 +208,16 @@ your branch so that it can be merged automatically.
 
 ## Improving Documentation
 
-TBD
+All MapStore components (i.e. plugins, actions, reducers etc) need to be properly documented in their JS documentation, please follow the required **JSDoc** syntax looking at the online documentation available in the [JSDoc web site](http://usejsdoc.org/index.html) for more details. 
+
+Each new feature/tool in MapStore needs to be documented also in the [User Guide](https://mapstore2.readthedocs.io/en/latest/user-guide/home-page/) in order to explain the involved functionalities and illustrate how it works. 
+
+All new front-end technologies, development procedures, best practices and guidelinees on the involved components in MapStore need to be properly documented too: the [Developers Guide](https://mapstore2.readthedocs.io/en/latest/developer-guide/) must be kept up-to-date for this.
```

Comment:
I don't think this is relevant for contributors. It's more a GeoSolutions internal practice.